benchmarks/benchmark_matmul.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
+import sparse
+
+
+class Matmul_Sparse:
```

Comment:
Maybe these two classes could also be parameterised.

Something like `dense_arg = [0, 1, None]`.